tests/meson.build

```diff
@@ -26,6 +26,24 @@ test_sources = [
 	'windows_compat.cpp'
 ]
 
+test_deps = []
+
+test_deps += dependency('nlohmann_json', fallback: ['json', 'nlohmann_json_dep'])
```

Comment:
You should just use the WrapDB wraps, which include the `fallback` information in Meson 0.54+ compatible format encoded in the wrap file, and this project needs that Meson version anyway.